docs/api-reference/testing-runtime.md

```diff
@@ -19,19 +19,24 @@ When writing tests, you are responsible for setting up and cleaning up your data
 
 ## Create Testing Runtime
 
-### createTestingRuntime(userClasses, \[configFilePath\])
+### createTestingRuntime(\[userClasses\], \[configFilePath\])
 ```typescript
-async function createTestingRuntime(userClasses: object[], configFilePath: string = dbosConfigFilePath): Promise<TestingRuntime>
+async function createTestingRuntime(userClasses: object[] | undefined = undefined, configFilePath: string = dbosConfigFilePath): Promise<TestingRuntime>
 ```
 
-Creates a testing runtime and loads user functions from provided `userClasses`.
+Creates a testing runtime and loads user functions from provided `userClasses`.  By default, all classes and dependencies are loaded and registered.
 Accepts an optional path to a [configuration file](./configuration.md), uses the default path (`dbos-config.yaml` in the package root) otherwise.
 
 For example, to create a runtime loading functions from the `Hello` class and using `test-config.yaml`:
 ```typescript
 testRuntime = await createTestingRuntime([Hello], "test-config.yaml");
 ```
 
+However, the defaults are generally sufficient as long as classes at least indirectly referenced from the test file:
```

Comment:
Maybe move the default first (commonly used), and then explain how to create a runtime for specific classes/config file.